frontend/src/components/UI/Stepper/ProductDetailsNavigation.vue

```diff
@@ -181,11 +65,13 @@ export default {
 			vOne,
 			vTwo,
 			modelValue,
+			drawerLeft: ref(true),
+			drawerRight: ref(true),
 		};
 	},
 	watch: {
 		modelValue(_oldModel, currentModel) {
-			if (currentModel === "Model") {
+			if (currentModel === "Modal") {
```

Comment:
Why it's not Model here ?

Reply:
ProductDetailsNavigation.vue file is no longer useful, I will delete it